slides/lectures/04/adc/slides.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: section
+---
+# ADC
+Analog to Digital Converter
+
+---
+---
+# Bibliography
+for this section
+
+**Raspberry Pi Ltd**, *[RP2040 Datasheet](https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf)*
+   - Chapter 4 - *Peripherals*
+     - Chapter 4.9 - *ADC and Temperature Sensor*
+       - Subchapter 4.9.1
+       - Subchapter 4.9.2
+       - Subchapter 4.9.5
+
+---
+layout: two-cols
+---
+# ADC
+sampling an analog signal to an array of values
+
+<style>
+.two-columns {
+    grid-template-columns: 3fr 5fr;
+}
+</style>
+
+| | | |
+|-|-|-|
+| *sampling rate* | Hz | the frequency at which a new sample is read |
+| *resolution* | bits | the number of bits used to store a sampled value |
+
+:: right ::
+
+<div align="center">
+<img src="/adc/sampling_values.svg" class="rounded w-150">
+</div>
+
+---
+---
+# Nyquist Theorem
+
+<div grid="~ cols-2 gap-5">
+
+<div>
+
+$$
+sampling_f >= 2 \times max_{f}
+$$
+
+*The **sampling frequency** has to be at least **two times higher** than the **maximum frequency** of the signal.*
+
```

Comment:
```suggestion

## Aliasing

Aliasing is the overlapping of frequency components. This overlap results in distortion or artifacts when the signal is reconstructed from samples which causes the reconstructed signal to differ from the original continuous signal.
```

Reply:
I would also add an aliasing image

Reply:
Do you have one?

Reply:
https://www.dataforth.com/media/g/aliasing-examples.png

Reply:
The image is not clear to me.

Reply:
This image shows a sine signal which is sampled at different sample rates below the Nyquist rate. The points are the samples and the different color signals are the results. The original signal and the result are overlapped to show the aliasing effect.

Reply:
There is a picture on the first slide, I added a text for it and explained aliasing.